Source/Util/Functions.c

```diff
@@ -0,0 +1,16 @@
+#include "Functions.h"
```

Comment:
> The entire PR is a mess. Truly. Commits are not properly squashed. Naming is completely off. Unneccessary changes at best. And i could go on. Also complaining that the escape character is wrong for windows while not actually fixing it in the PR.
> 
> The project DOES compile on windows AS IS. Without any additional other changes. In these changes you are FORCING users into Visual Studio and mingw or clang. Which they may not want to use or etc.
> 
> Considering that it builds on Windows AS IS this likely wont get merged unless there is a truly solid counter argument but even them im on the edge. Fix your build system. Not the project.
> 
> The main focus is Linux. And it will continue to be. Who wants to run game server on Windows ? You buy a cheapo VPS and run it on that. That being said ByteBit did amazing changes to make it work on Windows which were minimal and they continue to work like that. No further tweaks were needed. The recent builds failed because of pthreads. Which i just have to reinclude 1 library to fix that.
> 
> But lets see if it even builds on linux. (It doesnt).
> 
> But even then. As i said. It builds as is on Windows. And this just adds mess and complexity to something that already works.

If so, do you compile with MinGW?